doc/user-guide.md

```diff
@@ -0,0 +1,33 @@
+# User Guide
+## Discussions：
+
+<div align=center><href="https://sm.ms/image/DVrYCuwFlM4Pqni" target="_blank"><img src="https://s2.loli.net/2022/08/16/DVrYCuwFlM4Pqni.png" width="60%"></div>
+
+Welcome everyone to participate in the discussion about DP-GEN in the [discussion](https://github.com/deepmodeling/dpgen/discussions) module. You can ask for help, share an idea or anything to discuss here.
+Note: before you raise a question, please check TUTORIAL/FAQs and search history discussions to find solutions. 
+
+## Issue：
+
+<div align=center><href="https://sm.ms/image/Tu1dRX2L6zO5Yfq" target="_blank"><img src="https://s2.loli.net/2022/08/16/Tu1dRX2L6zO5Yfq.png" width="60%"></div>    
+      
+If you want to make a bug report or a request for new features, you can make an issue in the issue module. 
+
+<div align=center><href="https://sm.ms/image/FajXcmwbBJo34y1" target="_blank"><img src="https://s2.loli.net/2022/08/16/FajXcmwbBJo34y1.png" width="60%"></div>  
+  
+Here are the types you can choose. A proper type can help developer figure out what you need. Also, you can assign yourself to solve the issue. Your contribution is welcome!  
+
+Note: before you raise a question, please check TUTORIAL/FAQs and search history issues to find solutions. 
+      
+## Tutorials 
+
+Tutorials can be found [here](https://tutorials.deepmodeling.com/en/latest/Tutorials/DP-GEN/index.html).
+
+## Example for parameters
+      
+If you have no idea how to prepare a `PARAM` for your task, you can find examples of PARAM for different tasks in [examples](https://github.com/deepmodeling/dpgen/tree/master/examples).
+      
+For example, if you want to set specific template for LAMMPS, you can find an example [here](https://github.com/deepmodeling/dpgen/blob/master/examples/run/dp-lammps-enhance_sampling/param.json)
+      
+If you want to learn more about Machine parameters, please check [docs for dpdispatcher](https://docs.deepmodeling.com/projects/dpdispatcher/en/latest/)
+      
+## [Pull requests - How to contribute](./how-to-contribute.html)
```

Comment:
emmmm... The link does not work. I will try to find a solution if you fail to resolve it.

Reply:
I followed [answer-1444](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file). The relative link works in `Preview` but doesn't work in webpage.

Reply:
I push f86d56bd56031dbbb32c21d3149ed5944a12fd53 to resolve the issue. You can take a look.